src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
Do we need inheritance here? (`Lottery extends ColorSupplier`)

Reply:
If there exists an IS-A relationship between two classes, like for example Car IS-A Vehicle.